ports/nrf/README.md

```diff
@@ -98,11 +99,9 @@ note: On Linux it might be required to link SEGGER's `libjlinkarm.so` inside nrf
 
 ## DFU Targets
 
-    sudo apt-get install build-essential libffi-dev pkg-config gcc-arm-none-eabi git python python-pip
-    git clone https://github.com/adafruit/Adafruit_nRF52_Arduino.git
-    cd Adafruit_nRF52_Arduino/tools/nrfutil-0.5.2/
-    sudo pip install -r requirements.txt
-    sudo python setup.py install
+run follow command to install [adafruit-nrfutil](https://github.com/adafruit/Adafruit_nRF52_nrfutil) from PyPi
+
+    $ pip3 install adafruit-nrfutil --user
```

Comment:
Just change this to
```
$ pip3 install --user adafruit-nrfutil
```
which is the standard order of options followed by args (it doesn't matter to the command, but that's the usual order).